shared/src/actor/builtin.rs

```diff
@@ -56,6 +56,41 @@ impl Type {
     pub fn is_principal(&self) -> bool {
         self == &Type::Account || self == &Type::Multisig
     }
+
+    /// Returns the canonical (versioned) actor name, as exists in the manifest
+    pub fn name(&self) -> String {
+        match self {
+            Type::System => String::from("fil/v7/system"),
+            Type::Init => String::from("fil/v7/init"),
+            Type::Cron => String::from("fil/v7/cron"),
+            Type::Account => String::from("fil/v7/account"),
+            Type::Power => String::from("fil/v7/storagepower"),
+            Type::Miner => String::from("fil/v7/storageminer"),
+            Type::Market => String::from("fil/v7/storagemarket"),
+            Type::PaymentChannel => String::from("fil/v7/paymentchannel"),
+            Type::Multisig => String::from("fil/v7/multisig"),
+            Type::Reward => String::from("fil/v7/reward"),
+            Type::VerifiedRegistry => String::from("fil/v7/verifiedregistry"),
+        }
+    }
```

Comment:
Hm, I don't think we want any of this. The manifest in the bundle will contain a `Map<String, Cid>` where String is one of these values: https://github.com/filecoin-project/FIPs/pull/328/files#diff-409cddcb6e5c156f98898f7fe707d292b13d1d25be87bfe2363c39409c131e72R142-R152 (not the entire legacy Code CID, which doesn't make sense to have here).